test/help.test.js

```diff
@@ -45,6 +46,22 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  it('print right help message for each generator', function() {
+    process.env.SLC_COMMAND = 'loopback --help';
+    var names = ['acl', 'app', 'boot-script', 'datasource', 'export-api-def',
+    'middleware', 'model', 'property', 'relation', 'remote-method', 'swagger'];
+
+    names.forEach(function(name) {
+      var gen = givenGenerator(name, ['--help']);
+      var helpText = gen.help();
+      var helpFileName = 'loopback_' + name + '_help.txt';
+      var helpFilePath = '../fixtures/' + helpFileName;
+      var output = fs.readFileSync(helpFilePath, 'utf8');
+      assert.equal(output, helpText);
```

Comment:
The files in `intl` contain special directives for strong-globalize, e.g. `{{slc loopback}}`. Is it expected that the output of `gen.help()` includes them too and the return value of `gen.help()` has to be passed to strong-globalize?

As a user of this API, I would expect `get.help()` to return a text that was already processed/translated by strong-globalize.

Perhaps my understanding is wrong?